charters/wot-wg-2023-draft.html

```diff
@@ -491,6 +491,81 @@ <h3 id="w3c-coordination">W3C Groups</h3>
 
         <section>
           <h3 id="external-coordination">External Organizations</h3>
+          <p>
+            To succeed in establishing inter-platform standards,
+            W3C needs to coordinate with IoT alliances and standards development organizations.
+          </p>
+          <p>
+            List of new liaisons we want to form
+          </p>
+          <dl>
+            <dt> <a href="https://opcfoundation.org/">OPC Foundation</a>
+              <dd>
+                For coordination on the development of the OPC-UA Binding for the W3C Web of Things via the joint working group.
+                A Memorandum of Understanding (MoU) / Multi-Organization Cooperation Agreement (MOCA) will be signed.
+              </dd>
+            </dt>
+            <dt> <a href="https://industrialdigitaltwin.org/en/">Industrial Digital Twin Association</a>
+              <dd>
+                For coordination on the use of TDs and Binding Templates in the Asset Interface Description (AID) Submodel for the Asset Administration (AAS) Specification.
+                Additionally, the AAS Submodels can be used for Digital Nameplate and Digital Product Passport of products that the European Commission is demanding each product to supply.
+                Also see <a href="https://github.com/w3c/wot/issues/978#issuecomment-1359830278">the related GitHub comment</a>.
+              </dd>
+            </dt>
+            <dt> <a href="https://www.ashrae.org/">ASHRAE</a>
+              <dd>
+                For coordination on the development of the BACnet (ASHRAE 135-2020) Binding for the W3C Web of Things .
+              </dd>
+            </dt>
+            <dt><a href="https://github.com/Azure/opendigitaltwins-dtdl">Azure/Microsoft/OpenDigitalTwins DTDL</a></dt>
+            <dd>
+              For coordination on aligning the DTDL with the W3C WoT TD. 
+              NOTE: Microsoft is not an external organization so I am not sure how this should be put. The language itself is not limited to be used in Azure
+            </dd>
+          </dl>
+          <p>
+            List from previous charter to be verified. Add them to the list above if agreed.
+            There is also a wiki page linked but that page has not been updated since 2018.
+          </p>
+          
+          <p>The following list provides examples of organizations with which the WoT Working Group 
```

Comment:
Need to merge into one list.  Not mentioned: Digital Twin Consortium (DTC), IETF, CSA/Zigbee (Matter), OneDM, OGC, etc.  WE could also just list the "new" collaborations, "in addition to existing collaborations".

Reply:
could also use categories, e.g. "joint work", "review", etc.  We could also just reference a wiki page of active and planned collaborations.

Reply:
I have added all the orgs in the first comment. Goal of coordination for some of them is not clear and is indicated with a TBD.